script/add_sizes_to_versions

```diff
@@ -3,19 +3,13 @@
 ENV['RAILS_ENV'] ||= 'production'
 require_relative '../config/environment'
 
-def fog
-  $fog || Fog::Storage.new(:provider => 'Local',
-                           :local_root => Pusher.server_path('gems'))
-end
-
 Version.find_in_batches do |versions|
   versions.each do |version|
     print "sizing #{version.full_name}: "
-    file = fog.directories.get(Gemcutter.config['s3_bucket'] + '/gems') \
-      .files.get("#{version.full_name}.gem")
+    file = RubygemFs.instance.get("gems/#{version.full_name}.gem")
```

Comment:
I bet we don't need this script at all anymore. Could just remove it wholesale instead of updating.